.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -76,19 +76,19 @@ body:
     id: main-memory
     attributes:
       label: How much RAM is available to DMS explicitly?
-      description: Please provide the amount of memory that DMS can really use. This includes main memory (RAM), but also swap or zram. If you're using Kubernetes, provide the resource limit here if you set one.
+      description: Please provide the amount of memory that DMS can really use. This includes physical memory (RAM), but also virtual memory (like swap or zram).  Think about all possible resource restrictions on memory here.
```

Comment:
Accidentally reviewing a commit, so I can't provide a suggestion here.

> Think about all possible resource restrictions on memory here.

Perhaps "Consider any resource limits you may have set that would affect this."

Reply:
I like it.